pkg/verifier/verifier_test.go

```diff
@@ -22,40 +23,53 @@ import (
 	"sqlflow.org/sqlflow/pkg/parser"
 )
 
+func parseOneStatement(statement string) (*parser.SQLFlowSelectStmt, error) {
```

Comment:
Duplicated definition as in ir_generator_test.go

Reply:
Removed.